File: engine/b/src/main/java/net/crazyinvestor/engine_b/event/BithumbTickerEvent.java
package net.crazyinvestor.engine_b.event;

import net.crazyinvestor.engine_b.dto.response.BithumbWSTickerResponse;
import org.springframework.context.ApplicationEvent;

public final class BithumbTickerEvent extends ApplicationEvent {
    public BithumbTickerEvent(BithumbWSTickerResponse response) {
        super(response);
    }
}

File: engine/a/ticker_producer_engine/src/main/kotlin/net/crazyinvestor/ticker_producer_engine/deprecated/BithumbTickerEvent.kt
package net.crazyinvestor.ticker_producer_engine.deprecated

import net.crazyinvestor.ticker_producer_engine.dto.bithumb.response.BithumbWSTickerResponse
import org.springframework.context.ApplicationEvent

class BithumbTickerEvent(response: BithumbWSTickerResponse) : ApplicationEvent(response)